modules/snap_community_charts/charts.php

```diff
@@ -43,34 +48,35 @@
+  <div class="buttonset" id="baseline_buttons"><input checked="checked" id="baseline_cru" name="baseline" type="radio" value="cru32" /><label for="baseline_cru">CRU 3.2</label><input id="baseline_prism" name="baseline" type="radio" value="prism" /><label for="baseline_prism">PRISM</label></div>
 
-<div class="helpbuttons"><button id="dataset_help">More about scenarios</button></div>
+<div class="helpbuttons"><button id="baseline_help">Historical Baselines</button></div>
+<br/>
+<span style="font-size: 10px;">* Northwest Territory communities only available for CRU 3.2 baseline choice. </span>
+</div>
+</div>
 </div>
 
-<p>Inter–model variability</p>
-
-<div class="buttonset" id="variability_buttons"><input checked="checked" id="variability_off" name="variability" type="radio" value="0" /><label for="variability_off">Off</label> <input id="variability_on" name="variability" type="radio" value="1" /><label for="variability_on">On</label></div>
+<div class="col-md-6">
+<div class="inline-controls pull-left">
+<p>Representative Concentration Pathways </p>
 
-<div class="helpbuttons"><button id="variability_help">More about inter–model variability</button></div>
-</div>
+<div class="buttonset" id="scenario_buttons"><input id="scenario_rcp45" name="scenario" type="radio" value="rcp45" /><label for="scenario_rcp45">Low (RCP 4.5)</label> <input checked="checked" id="scenario_rcp60" name="scenario" type="radio" value="rcp60" /><label for="scenario_rcp60">Medium (RCP 6.0)</label> <input id="scenario_rcp85" name="scenario" type="radio" value="rcp85" /><label for="scenario_rcp85">High (RCP 8.5)</label></div>
 
+<div class="helpbuttons"><button id="dataset_help">Representative Concentration Pathways</button></div>
 </div>
 
-<div class="col-md-4">
-<div class="outlook-sidebar">
-<p><strong><a href="http://www.uaf.edu/ces/"><img alt="cooperative extension" class="plain pull-right" src="/sites/default/files/pictures/ces.jpg" style="width: 100px; height: 100px;" /></a>Created by SNAP in cooperation with&nbsp;<abbr title="Cooperative Extension Service">CES.</abbr> </strong></p>
-
-<p><strong>We offer climate outlooks for communities in this region:</strong></p>
-<img alt="alaska-canada" class="plain" src="/sites/default/files/pictures/akcanada_extent.gif" style="width: 279px; height: 113px;" />
+<div class="inline-controls pull-left">
+<p>Inter-model variability</p>
 
-<p><a class="btn btn-success" href="/">Return to SNAP home page</a></p>
+<div class="buttonset" id="variability_buttons"><input checked="checked" id="variability_off" name="variability" type="radio" value="0" /><label for="variability_off">Off</label> <input id="variability_overlay" name="variability" type="radio" value="1" /><label for="variability_overlay">Range</label><!--<input id="variability_exclusive" name="variability" type="radio" value="2" /><label for="variability_exclusive">Exclusive Range</label>--></div>
```

Comment:
This was an option for an exclusive range that Nancy and Tom didn't want in there, but I had commented it out in case the discussion looped back to them wanting it in there. Matt had been fairly excited about the Exclusive Range view, so I thought he might have had a swaying opinion on that being included, but I'll remove it now that it looks like that won't be the case.